src/WebJobs.Script.WebHost/Controllers/SwaggerController.cs

```diff
@@ -30,77 +30,46 @@ public SwaggerController(ISwaggerDocumentManager swaggerDocumentManager, WebScri
         [Route("admin/host/swagger/default")]
         public IHttpActionResult GetGeneratedSwaggerDocument()
         {
-            try
-            {
-                _traceWriter.Verbose(Resources.SwaggerGenerateDocument);
-                var swaggerDocument = _swaggerDocumentManager.GenerateSwaggerDocument(_scriptHostManager.HttpFunctions);
-                return Ok(swaggerDocument);
-            }
-            catch (Exception ex)
-            {
-                _traceWriter.Error(Resources.SwaggerGenerateError, ex);
-                return InternalServerError();
-            }
+            _traceWriter.Verbose(Resources.SwaggerGenerateDocument);
+            var swaggerDocument = _swaggerDocumentManager.GenerateSwaggerDocument(_scriptHostManager.HttpFunctions);
+            return Ok(swaggerDocument);
         }
 
         [HttpGet]
         [Route("admin/host/swagger")]
         public async Task<IHttpActionResult> GetSwaggerDocumentAsync()
         {
             IHttpActionResult result = NotFound();
-            try
+            if (_scriptHostManager.Instance.ScriptConfig.SwaggerEnabled)
```

Comment:
I guess I meant more philosophically speaking :D basically swagger content can be managed while not enabled for public consumption?